interop/java/src/main/java/com/microsoft/bond/interop/Main.java

```diff
@@ -48,16 +97,21 @@ public static void main(String[] args) throws IOException {
             ooe.l_b.add((byte) 33);
             ooe.v_s.add((short) 3);
             ooe.v_s.add((short) 33);
+            ooe.s_ui = new TreeSet<>();
```

Comment:
Why change the collection object? It's not nullable and hence not set to null before the setter is called. Is the intent to have a sorted order?